src/components/cms/BotBuilder/Preview/Preview.js

```diff
@@ -509,6 +509,10 @@ class Preview extends Component {
                                       })
                                     }
                                     onKeyDown={event => {
+                                      if (event.keyCode === 13) {
+                                        this.sendMessage();
+                                        this.setState({ message: '' });
+                                      }
```

Comment:
Please make the changes in onKeyDown function.

Reply:
What changes? @akshatnitd